src/App.js

```diff
@@ -1,10 +1,11 @@
+import { check } from 'prettier';
 import React, { useState } from 'react';
 import './App.css';
 
 import Board from './components/Board';
 
-const PLAYER_1 = 'X';
-const PLAYER_2 = 'O';
+const player1 = 'x';
+const player2 = 'o';
```

Comment:
We gave bad guidance here. This should have been
```js
const PLAYER_1 = 'x';
const PLAYER_2 = 'o';
```
since the tests were all written for lower case x and o. Looks like you got it straightened out yourself!